tools/ci/changed.sh

```diff
@@ -18,12 +18,18 @@
 
 patterns=()
 any_of=0
+all_of=0
 
+# --any-of: requires any of the changed files matche the given patterns
+# --all-of: requires all of the changed files matche the given patterns
```

Comment:
Is this the right name? From your usage, if the all-of matched, the job is not working. But for the any-of, if  matched, the job runs. Seems different logic.

Reply:
exclusion seems a better name?

Reply:
this script is nothing related to the Jenkins logic from the file name `tools/ci/changed.sh`, it just determines whether (all of/any of) the files matching the given patterns changed or not, the usage sounds something like `do all of the changed files match the given patterns`, or `does any of the changed files match the given patterns`, `exclusion` seems also ok to me but may be less semantic

Reply:
Jenkins just determine whether to run based on the result of this script

Reply:
Gor your point. It is ok to unchange.